tests/cnf/ran/gitopsztp/internal/tsparams/consts.go

```diff
@@ -83,6 +85,8 @@ const (
 	ZtpTestPathCustomSourceSearchPath = "ztp-test/custom-source-crs/search-path"
 	// ZtpKustomizationPath is the path to the kustomization file in the ztp test.
 	ZtpKustomizationPath = "/kustomization.yaml"
+	// ZtpTestPathDetachAISNO is the git path for the siteconfig operator detach AI SNO cluster instance test.
+	ZtpTestPathDetachAISNO = "ztp-siteconfigoperator-test/detach-ai-sno/"
```

Comment:
for consistency, the path should be `ztp-test/siteconfig-operator/detach-ai-sno` that way we have one directory for all ztp tests
also nit: this variable should be declared above ZtpKustomizationPath

Reply:
Okay, Noted for future use. I will remove this test directory structure definition here.
The reason for removal will be followed,

1. The test steps what I have developed here are common to detach-ai-sno and destach-ai-mno
2. Hence, i would like to approach more generic methodology here.
3. I will update the respective CI test profile to have and reflect the  test configuration directory structure required to verify detaching site resources at the root level.

Thanks,

Reply:
Done.